app/views/items/index.html.haml

```diff
@@ -47,10 +47,10 @@
   .row
     .column#menu
       - @categories.each do |category, category_items|
-        %h2.category-title= category.capitalize
+        %h2.category-title= category.capitalize if category.present?
```

Comment:
When you create an item and not specify a category, which is valid as of now. Should I change it?
@jhanggi